examples/cassandra/README.md

```diff
@@ -32,7 +33,19 @@ Documentation for other releases can be found at
 
 <!-- END MUNGE: UNVERSIONED_WARNING -->
 
-## Cloud Native Deployments of Cassandra using Kubernetes
+# Cloud Native Deployments of Cassandra using Kubernetes
+
+## Table of Contents
+
+  - [Prerequisites](#prerequisites)
+  - [tl;dr Quickstart](#tldr-quickstart)
+  - [Step 1: Create a Cassandra Service](#step-1-create-a-cassandra-service)
+  - [Step 2: Use a Replication Controller to create Cassandra node pods](#step-2-use-a-replication-controller-to-create-cassandra-node-pods)
+  - [Step 3: Scale up the Cassandra cluster](#step-3-scale-up-the-cassandra-cluster)
+  - [Step 4: Delete the Replication Controller](#step-4-delete-the-replication-controller)
+  - [Step 5: Use a DaemonSet instead of a Replication Controller](#step-5-use-a-daemonset-instead-of-a-replication-controller)
+  - [Step 6: Resource Cleanup](#step-6-resource-cleanup)
+  - [Seed Provider Source](#seed-provider-source)
 
 The following document describes the development of a _cloud native_
```

Comment:
While I didn't write this myself, I'd vote to keep the 1st pp as is (at least for this PR-- we can discuss further).  I don't think everyone knows what "cloud native" means, and it's an interesting and rather subtle concept.